core/src/main/java/org/elasticsearch/index/shard/StoreRecovery.java

```diff
@@ -162,10 +164,11 @@ void addIndices(
              * document-level semantics.
              */
             writer.setLiveCommitData(() -> {
-                final HashMap<String, String> liveCommitData = new HashMap<>(2);
+                final HashMap<String, String> liveCommitData = new HashMap<>(3);
```

Comment:
It looks like this should be four (it was already "wrong" at two). (It doesn't *really* matter, but if we are going to size the map, let's size it correctly and avoid a resize).

Reply:
good one. will fix.